laravelServer/server/resources/views/welcome.blade.php

```diff
@@ -154,6 +154,26 @@
                                 </div>
                             </div>
                         </div>
+						
+						<div class="p-6 border-t border-gray-200 dark:border-gray-700 md:border-l">
+                            <div class="flex items-center">
+                                <form method="post" action="{{ route('options') }}">
+                                    <div class="onoffswitch">
+                                        {{ csrf_field() }}
+                                        <input type="checkbox" name="status" class="onoffswitch-checkbox" id="placeholder7" tabindex="0" onchange="this.form.submit()" {{ $statuses->where('type', 6)->where('on', true)->isNotEmpty() ? 'checked' : '' }}>
+                                        <label class="onoffswitch-label" for="placeholder7"></label>
+                                        <input type="hidden" name="type" value="zuil">
```

Comment:
currently type is a smallInt type of pgsql. So if we try to upload this our database will say wtf do you mean VARCHAR. wtf even is that. I want an int between -32,768 and 32,767